src/ucs/arch/x86_64/cpu.c

```diff
@@ -363,6 +363,14 @@ ucs_cpu_model_t ucs_arch_get_cpu_model()
        }
     }
 
+    if (family == 0x17) {
+        switch (model) {
+        case 0x29:
+            return UCS_CPU_MODEL_AMD_NAPLES;
+        case 0x31:
+            return UCS_CPU_MODEL_AMD_ROME;
+        }
+    } 
```

Comment:
add a blank line before `return`

Reply:
I actually am not sure if it's necessary because the switch case above it has not blank lines. Is it required by the UCX coding style? If so, I can add lines for each switch case return.

Reply:
@paklui feel free to ignore